data/sql_incremental_aggregates/prepare_schedule_a_aggregate_employer.sql

```diff
@@ -14,60 +14,17 @@ and is_individual(contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text, c
 group by cmte_id, cycle, employer
 ;
 
-alter table ofec_sched_a_aggregate_employer_tmp add column idx serial primary key;
+alter table ofec_sched_a_aggregate_employer_tmp add column idx serial;
+alter table ofec_sched_a_aggregate_employer_tmp add constraint uq_cmte_id_cycle_employer unique (cmte_id, cycle, employer);
 
 -- Create indices on aggregate
-create index on ofec_sched_a_aggregate_employer_tmp (cmte_id, idx);
-create index on ofec_sched_a_aggregate_employer_tmp (cycle, idx);
-create index on ofec_sched_a_aggregate_employer_tmp (employer, idx);
-create index on ofec_sched_a_aggregate_employer_tmp (total, idx);
-create index on ofec_sched_a_aggregate_employer_tmp (count, idx);
+create index on ofec_sched_a_aggregate_employer_tmp (cmte_id);
+create index on ofec_sched_a_aggregate_employer_tmp (cycle);
+create index on ofec_sched_a_aggregate_employer_tmp (employer);
+create index on ofec_sched_a_aggregate_employer_tmp (total);
+create index on ofec_sched_a_aggregate_employer_tmp (count);
 create index on ofec_sched_a_aggregate_employer_tmp (cycle, cmte_id);
 
 -- Remove previous aggregate and rename new aggregate
 drop table if exists ofec_sched_a_aggregate_employer;
 alter table ofec_sched_a_aggregate_employer_tmp rename to ofec_sched_a_aggregate_employer;
-
--- Create update function
-create or replace function ofec_sched_a_update_aggregate_employer() returns void as $$
-begin
-    with new as (
-        select 1 as multiplier, cmte_id, rpt_yr, contbr_employer, contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text, contbr_id
-        from ofec_sched_a_queue_new
-    ),
-    old as (
-        select -1 as multiplier, cmte_id, rpt_yr, contbr_employer, contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text, contbr_id
-        from ofec_sched_a_queue_old
-    ),
-    patch as (
-        select
-            cmte_id,
-            rpt_yr + rpt_yr % 2 as cycle,
-            contbr_employer as employer,
-            sum(contb_receipt_amt * multiplier) as total,
-            sum(multiplier) as count
```

Comment:
I just wanted to clarify, we're not losing these aggregate calculations based on the changes below, correct?  It doesn't appear that way but I just wanted to be sure!

Reply:
We shouldn't be, based on my understanding of the logic.